src/main/java/twilightforest/entity/EntityTFGoblinKnightUpper.java

```diff
@@ -218,7 +219,7 @@ public void updateRidden() {
 //            this.renderYawOffset = ((EntityLiving)this.ridingEntity).renderYawOffset;
 //        }
         super.updateRidden();
-        if (this.ridingEntity != null) {
+        if (this.ridingEntity instanceof EntityLivingBase) {
```

Comment:
Why `EntityLivingBase` rather than `EntityLiving`?

Reply:
That's the way wizardry checks if a spell should affect a living entity.
I'm not familiar with all its effects and why it would require such so I went the safe way.

Reply:
then `ridingEntity` should be casted to `EntityLivingBase` as well

Reply:
Done